nautobot/project-static/css/base.css

```diff
@@ -13,17 +13,11 @@ body {
     padding-top: 70px;
 }
 .wrapper {
-    min-height: 100%;
-    height: auto !important;
-    margin: 0 auto -48px; /* the bottom margin is the negative value of the footer's height */
-    padding-bottom: 30px;
+    min-height: calc(100vh - 110px);
 }
 .navbar-brand {
     padding: 12px 15px 8px;
 }
-.footer, .push {
-    height: 48px; /* .push must be the same height as .footer */
-}
```

Comment:
We either need to keep `.push` or update styling for `BANNER_BOTTOM`:

New appearance
<img width="1167" alt="Screen Shot 2022-10-11 at 13 10 28" src="https://user-images.githubusercontent.com/31187/195156774-63a869e6-7e95-4353-a6ca-7cff4011e679.png">

Previous appearance
<img width="1125" alt="Screen Shot 2022-10-11 at 13 12 16" src="https://user-images.githubusercontent.com/31187/195156885-69bc8e5b-b7ec-4f7b-b594-48036e411a5f.png">